plugins/uefi/fu-uefi-bootmgr.h

```diff
@@ -13,8 +13,15 @@
 
 G_BEGIN_DECLS
 
-gboolean	 fu_uefi_bootmgr_bootnext	(const gchar	*esp_path,
-						 GError		**error);
+typedef enum {
+	FU_UEFI_BOOTMGR_FLAG_NONE	= 0,
+	FU_UEFI_BOOTMGR_FLAG_USE_SHIM	= 1 << 0,
```

Comment:
Similar to other comments I think that `FU_UEFI_BOOTMGR_FLAG_USE_SHIM` should be renamed to take into account shim is only used for secure boot.  Proposal:
`FU_UEFI_BOOTMGR_FLAG_USE_SHIM_FOR_SB`

Reply:
Fixed.